### ISSUE TITLE
Static tasks include when using import_role - no tasks included

### ISSUE THREAD

**Issue by gpr95** [confluentinc/cp-ansible#524]
**Describe the issue**
When using import_role from 6.0.1-post version we I got:
```
[DEPRECATION WARNING]: Included file
'/mypath/tests/tasks/failure_handling.yml' not found,
however since this include is not explicitly marked as 'static: yes', we will
try and include it dynamically later. In the future, this will be an error
unless 'static: no' is used on the include task. If you do not want missing
includes to be considered dynamic, use 'static: yes' on the include or set the
global ansible.cfg options to make all includes static for tasks and/or
handlers. This feature will be removed in version 2.12. Deprecation warnings
can be disabled by setting deprecation_warnings=False in ansible.cfg.
```

 static include of tasks are not used because of relative path in for example:

```
- name: Fetch Files for Debugging Failure
  include: tasks/failure_handling.yml
  vars:
    service_name: "{{zookeeper_service_name}}"
    config_file: "{{zookeeper.config_file}}"
  when: status.failed or quorum.failed
```

**To Reproduce**
Create role with:
```
- name: use common
  import_role:
    name: confluent.common
```

**Expected behaviour**
All tasks should be included with {{ role_path }} usage without relative paths

**Environment (please complete the following information):**
 - OS: any OS
 - CP-Ansible Branch: 6.0.1-post
 - Ansible Version 2.9.12

**Comment by erikgb** [comment]
@gpr95 I think your problem might have been fixed on the 6.1.x branch. I was struggeling with similar issues as you, and prepared a PR to mend it. But it turned out that it was not as simple as it appeared. After some investigation by @JumaX summarized in https://github.com/confluentinc/cp-ansible/pull/449#issuecomment-716622078, we seems to hit an open Ansible bug here: https://github.com/ansible/ansible/issues/20493

**Comment by erikgb** [comment]
@gpr95 Are you able to verify if this bug is still present or not?

**Comment by rohit2b** [comment]
This should be fixed on 6.1.x. Closing this issue, please re-open if you find this bug persists.
